snprc_ehr/resources/source_queries/create_v_charge_account.sql

```diff
@@ -46,6 +48,7 @@ ALTER VIEW [labkey_etl].[v_charge_account] AS
   FROM dbo.charge_account AS ca
     LEFT OUTER JOIN dbo.TAC_COLUMNS AS tc ON tc.object_id = ca.object_id
 	LEFT OUTER JOIN dbo.valid_charge_by_species AS vcs ON ca.charge_id = vcs.charge_id
+    LEFT OUTER JOIN dbo.prd_cost_account AS pca on pca.account_id = ca.account_id
```

Comment:
I think LEFT OUTER JOIN is same as OUTER JOIN?

Reply:
You would be wrong 🙂🙂

Reply:
https://learn.microsoft.com/en-us/sql/relational-databases/performance/joins?view=sql-server-ver16

- left outer join
- right outer join
- full outer join
-

Reply:
You're probably thinking of a "left join" being the same as a "left outer join"

Reply:
Yes Terry. Correct. LEFT JOIN being same as LEFT OUTER JOIN and we don't usually use 'OUTER' when the same can be done using LEFT JOIN.

Reply:
I always use outer, and I always use inner. But, that's me - I'm an old bugger.